src/main/java/org/terasology/gf/PlantGrowingSystem.java

```diff
@@ -64,7 +65,7 @@ public void updatePlant(DelayedActionTriggeredEvent event, EntityRef plant, Livi
             PerformanceMonitor.startActivity("GrowingFlora - Updating plant");
             try {
                 PlantGrowthDefinition plantDefinition = plantRegistry.getPlantGrowthDefinition(plantComponent.type);
-                Long updateDelay = plantDefinition.requestedUpdatePlant(worldProvider, environmentSystem, blockEntityRegistry, plant);
+                Long updateDelay = plantDefinition.requestedUpdatePlant(worldProvider, new EnvironmentLocalParameters(environmentSystem, blockComponent.getPosition()), blockEntityRegistry, plant);
```

Comment:
This whole system will only be initialized if all it's dependencies can be met. If ClimateConditions is not loaded, we'll see logs like the following, and the whole feature is not present. 

```
22:47:03.042 [main] WARN  o.t.e.core.ComponentSystemManager - Skip system GrowingFlora:SaplingInitializeSystem for loading - possibly missing optional dependencies
22:47:03.045 [main] WARN  o.t.e.core.ComponentSystemManager - Skip system GrowingFlora:PlantGrowingSystem for loading - possibly missing optional dependencies
```

I think this is fine for now, but we can definitely improve on this, e.g., by basing those plant definition on something else if the `environmentSystem` is not present.

I think this is something for another PR, though. Maybe Terasology/SimpleFarming#120 can give some reference.

Reply:
I actually fixed this right before this comment. Those systems are needed for plants to grow, so skipping them isn't great.